CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 This document outlines major changes between releases.
 
+## [0.22.0] - 2022-07-25
+
+Tree service support
+
+### Fixed
+- Error logging (#450)
+- Default bucket location constraint (#463)
+- Suspended versioning status (#462)
+- CodeQL warnings (#489, #539)
```

Comment:
Should mention https://github.com/nspcc-dev/neofs-s3-gw/issues/522 too I suppose